docs/bokeh/source/docs/user_guide/examples/categorical_heatmap_periodic.py

```diff
@@ -33,7 +33,7 @@
 p = figure(width=900, height=500, title="Periodic table (omitting LA and AC series)",
            x_range=groups, y_range=list(reversed(periods)), toolbar_location=None, tools="hover")
 
-p.rect("group", "period", 0.95, 0.95, source=source, fill_alpha=0.6, legend_field="metal",
+a = p.rect("group", "period", 0.95, 0.95, source=source, fill_alpha=0.6, legend_field="metal",
```

Comment:
Most other examples tend to use `r` (short for "renderer") in cases like this, is there a reason to use `a` here instead?

Reply:
As `r` is used in other places too, hover renderer gets applied to the last `r`, which in this case represents the atomic mass. So it will display the tooltip only when you hover over the atomic mass text and not when you hover over other places in the rect glyph. So that's the reason

Reply:
Like in the example, I could change this file a little bit so that I have to only name rect glyph as `r`
![image](https://user-images.githubusercontent.com/71313814/191580490-f4ffef2b-b885-42e7-b811-8455f591b1a4.png)